iothub/device/tests/Transport/Amqp/MockableAmqpUnit.cs

```diff
@@ -13,17 +14,19 @@ namespace Microsoft.Azure.Devices.Client.Test.Transport
 {
     internal class MockableAmqpUnit : AmqpUnit
     {
+        private static IotHubClientAmqpSettings s_transportSettings = new();
```

Comment:
Need to explicitly initialize AMQP for this mock, and I changed the init below to all refer to the same instance.